tools/deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+if [[ "$1" == "" ]]; then
+	echo "No branch name provided"
+	exit 1
+fi
+
+echo "Deploying branch $1"
+
+echo 'cd pure && git fetch origin && git checkout '$1' && export NODE_ENV="development" && echo $NODE_ENV && npm install && export NODE_ENV="production" && cd src/submodules/know && git checkout master && git pull && npm install && cd ../../.. && npm run build && sudo restart pure && tail -f logs/now.log' | ssh ubuntu@pure.heyneighbor.chat
```

Comment:
Can you take the SSH credentials as $2 (`ssh $2`) and give it while typing the command: `./release.sh ubuntu@bel.ng`.

That makes it more generic.